web/src/stream_settings_ui.ts

```diff
@@ -310,16 +343,16 @@ export function update_settings_for_subscribed(slim_sub) {
     update_empty_left_panel_message();
 }
 
-export function show_active_stream_in_left_panel() {
+export function show_active_stream_in_left_panel(): void {
     const selected_row = hash_parser.get_current_hash_section();
 
     if (Number.parseFloat(selected_row)) {
-        const $sub_row = stream_ui_updates.row_for_stream_id(selected_row);
+        const $sub_row = stream_ui_updates.row_for_stream_id(Number.parseFloat(selected_row));
```

Comment:
Should we just do `const selected_row = Number.parseFloat(hash_parser.get_current_hash_section());` above.

Reply:
I like that!